podvm/files/etc/systemd/system/api-server-rest.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=CDH API Server Rest Service
+BindsTo=netns@podns.service
+After=network.target confidential-data-hub.service netns@podns.service
+
+[Service]
+Type=simple
+NetworkNamespacePath=/run/netns/podns
+ExecStart=/usr/local/bin/api-server-rest --features all 
+
+Restart=always
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
API server when manually started fails as:

```bash
Sep 15 02:40:48 podvm-nginx-secret-pod-eb58589a systemd[1]: Started CDH API Server Rest Service.
Sep 15 02:40:48 podvm-nginx-secret-pod-eb58589a api-server-rest[1363]: Starting API server on 127.0.0.1:8006 with features all
Sep 15 02:40:48 podvm-nginx-secret-pod-eb58589a api-server-rest[1363]: Error: Nix error: ENOENT: No such file or directory
Sep 15 02:40:48 podvm-nginx-secret-pod-eb58589a api-server-rest[1363]: Caused by:
Sep 15 02:40:48 podvm-nginx-secret-pod-eb58589a api-server-rest[1363]:     ENOENT: No such file or directory
Sep 15 02:40:48 podvm-nginx-secret-pod-eb58589a systemd[1]: api-server-rest.service: Main process exited, code=exited, status=1/FAILURE
Sep 15 02:40:48 podvm-nginx-secret-pod-eb58589a systemd[1]: api-server-rest.service: Failed with result 'exit-code'.
Sep 15 02:40:49 podvm-nginx-secret-pod-eb58589a systemd[1]: api-server-rest.service: Scheduled restart job, restart counter is at 5.
Sep 15 02:40:49 podvm-nginx-secret-pod-eb58589a systemd[1]: Stopped CDH API Server Rest Service.
Sep 15 02:40:49 podvm-nginx-secret-pod-eb58589a systemd[1]: api-server-rest.service: Start request repeated too quickly.
Sep 15 02:40:49 podvm-nginx-secret-pod-eb58589a systemd[1]: api-server-rest.service: Failed with result 'exit-code'.
Sep 15 02:40:49 podvm-nginx-secret-pod-eb58589a systemd[1]: Failed to start CDH API Server Rest Service.
```

Reply:
I see this too and I was hoping that this was related to the CDH not starting, but that might be wishful thinking, so I've added the info to the issue so that Arron/Ding have a chance to see it. Thanks for nudging me.